cardinal/evm/server.go

```diff
@@ -107,6 +110,26 @@ func (s *srv) SendMsg(ctx context.Context, msg *routerv1.MsgSend) (*routerv1.Msg
 		return nil, err
 	}
 	// add transaction to the world queue
-	s.txh.AddTransaction(itx.ID(), tx, nil)
+	s.world.AddTransaction(itx.ID(), tx, nil)
 	return &routerv1.MsgSendResponse{}, nil
 }
+
+func (s *srv) QueryShard(ctx context.Context, req *routerv1.QueryShardRequest) (*routerv1.QueryShardResponse, error) {
```

Comment:
Im cautious of using the word "query" here since it means something in ECS. Maybe we should rename this to read for consistency?

Reply:
yeah, i was veering either way, cause `Read` is quite specific naming to Cardinal, and the precompile is eventually generalized to many different game shards, where i imagine query will be the default word for these types of things. 

the more i think about this, the more i want to rename our read system to query honestly lol

Reply:
grrr, yeah i guess query makes more sense